Client/Frontend/Widgets/PhotonActionSheetProtocol.swift

```diff
@@ -177,6 +178,10 @@ extension PhotonActionSheetProtocol {
                 _ = self.profile.history.addPinnedTopSite(site).value
             }
         }
+
+        let sendToDevice = PhotonActionSheetItem(title: Strings.AppMenuSendToDeviceTitleString, iconString: "menu-Send-to-Device") { action in
+            presentSendTo()
```

Comment:
all the code in the BVC can move here 😄 
`presentableVC` is actually the BVC. so you can simply do `presentableVC.present` 

you need to double check if presentableVC conforms to the delegate protocols used by the two VCs 
`           guard let bvc = presentableVC as? InstructionsViewControllerDelegate & ClientPickerViewControllerDelegate else {
                return
            }
`